helix-core/src/test/java/org/apache/helix/cloud/event/HelixTestCloudEventHandler.java

```diff
@@ -0,0 +1,16 @@
+package org.apache.helix.cloud.event;
+
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+
+public class HelixTestCloudEventHandler extends CloudEventHandler {
+  private static final int TIMEOUT = 900;  // second to timeout
+  private static ExecutorService executorService = Executors.newSingleThreadExecutor();
+
+  @Override
+  public void registerCloudEventListener(CloudEventListener listener) {
+    super.registerCloudEventListener(listener);
+    System.out.println("in child register");
+  }
+
+}
```

Comment:
Could we test unregister as well? My imagination would be we disconnect the zk helix manager, and the listener should be automatically unregistered. It is missing in the current test, could you help add this test? Thanks!

Reply:
Thanks for suggestion. Test added.